src/pages/Environments.js

```diff
@@ -547,6 +413,9 @@ export const Environments = ({ nav = undefined }) => {
 
   const renderErrorApps = app => {
     const convertedAppStatus = getComputeStatusForDisplay(app.status)
+    if (convertedAppStatus !== 'Error' && app.unsupportedWorkspace) {
```

Comment:
If there is an Error message from Leo, let's display that because it might be useful.